oemoflex/preprocessing.py

```diff
@@ -330,7 +371,7 @@ def create_wind_onshore_profiles(data_raw_path, data_preprocessed_path):
     )
 
     wind_onshore_profile_df = combine_profiles(
-        raw_wind_onshore_profile_paths, 'el-wind-onshore-profile'
+        raw_wind_onshore_profile_paths, 'wind-onshore-profile'
```

Comment:
Same here and some lines down: Can we use this already-defined strings from a csv file?